src/service-worker.ts

```diff
@@ -98,97 +86,76 @@ export async function generateServiceWorkerConfig(
   options = Object.assign({}, options);
   const project = options.project;
   const buildRoot = options.buildRoot;
-  const swPrecacheConfig: SWConfig =
-      Object.assign({}, options.swPrecacheConfig);
+  const workboxConfig: WorkboxConfig =
+    Object.assign({}, options.workboxConfig);
 
   const depsIndex = await project.analyzer.analyzeDependencies;
-  let staticFileGlobs = Array.from(swPrecacheConfig.staticFileGlobs || []);
+  let staticFileGlobs = Array.from(workboxConfig.globPatterns || []);
   const precachedAssets = (options.bundled) ?
-      getBundledPrecachedAssets(project) :
-      getPrecachedAssets(depsIndex, project);
+    getBundledPrecachedAssets(project) :
+    getPrecachedAssets(depsIndex, project);
 
   staticFileGlobs = staticFileGlobs.concat(precachedAssets);
   staticFileGlobs = staticFileGlobs.map((filePath: string) => {
     if (filePath.startsWith(project.config.root)) {
       filePath = filePath.substring(project.config.root.length);
     }
-    return path.join(buildRoot, filePath);
+    return removeLeadingSlash(filePath);
   });
 
-  if (swPrecacheConfig.navigateFallback === undefined) {
+  if (workboxConfig.globDirectory === undefined) {
+    workboxConfig.globDirectory = buildRoot;
+  }
+
+  if (workboxConfig.importScripts === undefined) {
+    // Map all application routes to the entrypoint.
+    workboxConfig.importScripts =
+      ['https://storage.googleapis.com/workbox-cdn/releases/3.0.0/workbox-sw.js'];
```

Comment:
The `workbox-build` node module exports a `getModuleUrl()` function, which will return the current CDN URL. I'd switch to that instead of hardcoding.

https://github.com/GoogleChrome/workbox/blob/ed2a847baf34c17c0f019d11cdfb1464fe3a9e19/packages/workbox-build/src/index.js#L86

(This is another method that is currently only used internally in the Webpack plugin, but it is supported.)

Reply:
TIL! Done